gsadjust/pyqt_models.py

```diff
@@ -1616,39 +1632,32 @@ class DatumTableModel(QtCore.QAbstractTableModel):
     Model to store Datums, shown on the adjust tab.
     """
 
+    _headers = { # As map, so do not need to be kept in order with the above.
+        DATUM_STATION: 'Station',
+        DATUM_G: 'g',
+        DATUM_SD: 'Std. dev.',
+        DATUM_DATE: 'Date',
+        DATUM_TIME: 'Time (UTC)',
+        MEAS_HEIGHT: 'Meas. height',
+        GRADIENT: 'Gradient', 
+        DATUM_RESIDUAL: 'Residual', 
+        N_SETS: '# sets',
+    }
+
+    signal_adjust_update_required = QtCore.pyqtSignal()
```

Comment:
I've moved these signal definitions above the `__init__`. It doesn't affect behaviour, however having them under a method leaves you open to accidentally indenting them and them no longer working.